VMBackup/main/parameterparser.py

```diff
@@ -34,6 +34,8 @@ def __init__(self, protected_settings, public_settings, backup_logger):
         self.customSettings = None
         self.snapshotTaskToken = ''
         self.includedDisks = None
+        self.settings = None
+        self.wellKnownSettingFlags = {'isSnapshotTTLEnabled','UseMCCFForLAD','UseMCCFToFetchDSASForAllDisks'}
```

Comment:
Same comment about camel vs pascal casing.

Could we initiate self.wellKnownSettingFlags to {CommonVariables.isSnapshotTTLEnabled, CommonVariables.UseMCCFForLAD, CommonVariables.UseMCCFToFetchDSASForAllDisks}?